### PR TITLE
Interactive Prompt Attributions

### LINKED ISSUE #23584: Prompts: Attribution External Links
### Details:
1. Launch WPiOS
2. Press on `My Site`
3. Press over the `From DayOne` attribution label

### Expected:
DayOne's landing site should be open onscreen.

### Current behavior:
We're not currently handling tap events over the Attribution UI.

### PR DESCRIPTION
**Pull request by jleandroperez**
Fixes #23584

## Context:
We're updating the Prompts UI so that, on tap, the DayOne landing site is presented (when applicable).

Reference: 108-gh-bloom/Firebird-Issues

cc @kean  @jkmassel Thanks in advance gentleman!!

## To test:
1. Launch WPiOS
2. Press on My Site
3. Press over the From DayOne attribution label

- [ ] Verify Safari is presented onscreen
- [ ] Verify that the DayOne landing site is rendered

## Regression Notes
1. Potential unintended areas of impact
None!

4. What I did to test those areas of impact (or what existing automated tests I relied on)
N/A

5. What automated tests I added (or what prevented me from doing so)
Not covered by automation

PR submission checklist:

- [ ] I have completed the Regression Notes.
- [ ] I have considered adding unit tests for my changes.
- [ ] I have considered adding accessibility improvements for my changes.
- [ ] I have considered if this change warrants user-facing release notes and have added them to `RELEASE-NOTES.txt` if necessary.

Testing checklist:
- [ ] WordPress.com sites and self-hosted Jetpack sites. < **Non applicable**
- [x] Portrait and landscape orientations.
- [x] Light and dark modes.
- [x] Fonts: Larger, smaller and bold text.
- [x] High contrast.
- [x] VoiceOver.
- [x] Languages with large words or with letters/accents not frequently used in English.
- [x] Right-to-left languages. (Even if translation isn’t complete, formatting should still respect the right-to-left layout)
- [x] iPhone and iPad. 
- [x] Multi-tasking: Split view and Slide over. (iPad)

## Screenshots
Light | Dark
-|-
 | 

### Demo
Simulator Screen Recording - iPhone 15 Pro - 2024-09-11 at 17 11 18

### CODE REVIEW

**Comment by dangermattic** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true" data-kind="Warning">
          1 Warning
      </th>
     </tr>
  </thead>
  <tbody>
    <tr>
      <td>:warning:</td>
      <td data-sticky="false">PR is not assigned to a milestone.</td>
    </tr>
  </tbody>
</table>

<p align="right" data-meta="generated_by_pr-check">
  Generated by :no_entry_sign: <a href="https://danger.systems/">Danger</a>
</p>

File: WordPress/Classes/ViewRelated/Blog/Blog Dashboard/Cards/Prompts/BloggingPromptsAttribution.swift [hunk 1]
```diff
@@ -45,8 +63,12 @@ enum BloggingPromptsAttribution: String {
...
             .foregroundColor: UIColor.label,
         ]
-        static let iconSize = CGSize(width: 18, height: 18)
-        static let dayOneIcon = UIImage(named: "logo-dayone")?.resized(to: Constants.iconSize)
+        static let dayOneIconSize = CGSize(width: 18, height: 18)
+        static let dayOneIcon = UIImage(named: "logo-dayone")?.resized(to: Constants.dayOneIconSize)
+        static let dayOneURL = URL(string: "https://dayoneapp.com/?utm_source=jetpack&utm_medium=prompts")
+
+        static let linkIconSize = CGSize(width: 12, height: 12)
+        static let linkIcon = UIImage.gridicon(.external, size: Constants.linkIconSize)
```
Comment (kean) [inline WordPress/Classes/ViewRelated/Blog/Blog Dashboard/Cards/Prompts/BloggingPromptsAttribution.swift]: (nit) I'd suggest using an SF symbol like "link" or one of the image from the WordPress Design Library. We are slowly removing `Gridicon` usages in the app.

**Review by kean — APPROVED** [review]
Hey, 👋 
The code looks good, approving.

**Comment by jleandroperez** [comment]
✅ Switched Gridicons > SF Icons + screenshots updated

Thanks for reviewing this one Alex!!

**Pull request merged**